internal/cri/server/images/image_pull.go

```diff
@@ -347,6 +385,8 @@ func (c *CRIImageService) createOrUpdateImageReference(ctx context.Context, name
 }
 
 // getLabels get image labels to be added on CRI image
+// If by any chance SandboxImage was not set(e.g. CRI config was only partially set),
```

Comment:
Is this added comment related?

Reply:
No, perhaps from a previous rebase. Removed.